docs/HISTORY.txt

```diff
@@ -4,7 +4,8 @@ Changelog
 4.0.6 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Reorder HEAD elements so that CSS comes before JavaScript - which is correct and proper 
+  This solves an issue in ftw.colorbox where overlays are not always loaded. [djowett-ftw]
```

Comment:
@djowett-ftw Since I installed the change already on the DEV System, this is not true. The first line is `Reorder HEAD elements so that CSS comes before JavaScript - which is correct and proper`

Reply:
> Question:
> 
> Would it be enough to just swap line 52 and 54? I'm not sure about the impact of the other changes.

There's just one CSS and no JS in #portal-top for izug, so no that wouldn't be enough.
The extraction (and re-ordering) of the new line 47 from 41 is the main bit, but it's proper to put #portal-top script after that too (and that's a no-op for izug I believe)

Reply:
> @djowett-ftw Since I installed the change already on the DEV System, this is not true. The first line is `Reorder HEAD elements so that CSS comes before JavaScript - which is correct and proper`

Good spot!  Done.

Reply:
> > Question:
> > Would it be enough to just swap line 52 and 54? I'm not sure about the impact of the other changes.
> 
> There's just one CSS and no JS in #portal-top for izug, so no that wouldn't be enough.
> The extraction (and re-ordering) of the new line 47 from 41 is the main bit, but it's proper to put #portal-top script after that too (and that's a no-op for izug I believe)

@maethu was my argument convincing?